src/routes/request/index.ts

```diff
@@ -14,9 +14,20 @@ jsonRPCRouter.method("eth_debugUserOperation", async (ctx: any) => {
 // Add other rpc methods here as mentioned above
 jsonRPCRouter.method("eth_config", async (ctx: any) => {
     ctx.body = {
-        supportedNetworks,
     };
     ctx.status = 200;
 });
 
+
+jsonRPCRouter.method("eth_supportedEntryPoints", async (ctx: any) => {
+    const networkId = ctx.params.networkId;
+    const data = [{
+        label: "EntrypointV6",
+        value: networkConfig[networkId].entryPointV6
```

Comment:
Check if networkConfig[networkId].entryPointV6 exists.. then return the label and value in response.